.travis/run_tests

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+set -e
+
+# For debug only:
+# export DEBUG=true
+# set -x
+# set -v
+
+BASEDIR="$(cd "$(dirname "${BASH_SOURCE[0]}")/.." && pwd)"
+
+# Travis runs do not rely on Vagrant
+export USE_VAGRANT=false
+export DOTBOT_EXEC="${BASEDIR}/bin/dotbot"
+
+cd "${BASEDIR}"
+. "test/driver-lib.bash"
+
+travis_initialize() {
+    echo "initializing."
+    tests_run=0
+    tests_passed=0
+    tests_failed=0
+    tests_total="${1}"
+    local plural="" && [ "${tests_total}" -gt 1 ] && plural="s"
+    printf -- "running %d test%s...\n\n" "${tests_total}" "${plural}"
+}
+
+travis_cleanup() {
```

Comment:
Clearly not as foolproof as the Vagrant approach, but in that specific context, that seems to be good enough. A more complex method could be used if it was really necessary (e.g. taking snapshots of the home directory).